src/main/java/randoop/generation/ForwardGenerator.java

```diff
@@ -38,6 +41,39 @@
 /** Randoop's forward, component-based generator. */
 public class ForwardGenerator extends AbstractGenerator {
 
+  /**
```

Comment:
This is a lot of fields, with unclear documentation (such as just "Hyperparameter for decreasing weights.") and undocumented relationship to the rest of the class.  Maybe the structure could be clarified by making a struct for all of these?  That could also make it easier to turn on and off the entire feature.